tools/gen-intrinsics-specs.py

```diff
@@ -800,9 +807,11 @@ def get_intrinsics_db(path):
     classification_map = get_classification_map(cli_args.classification)
     intrinsics_db = get_intrinsics_db(cli_args.intrinsic_defs)
     doc_template = read_template(cli_args.template)
-    intrinsic_table = process_db(intrinsics_db, classification_map)
+    output_file = cli_args.outfile
+    table_format = cli_args.format
```

Comment:
This two variables are used only as parameters of `process_db`. I think you can get rid of them and pass these values directly in the invocation of `process_db`